.circleci/config.yml

```diff
@@ -3490,6 +3500,27 @@ workflows:
             - slack
             - circleci-repo-readonly-authenticated-github-token
 
+  # Weekly cache warming for upgrade tests.
+  # Runs at 01:00 UTC on Sundays to warm the Foundry RPC cache for the new week's
```

Comment:
What happens to other runs that fall between `sundayMidnightUTC()` and 1:00 UTC on Sundays?

Will they just end up being a bit slower because they are warming the cache prior to this job? I think that's probably OK, just trying to understand.

Reply:
Yes, that's the case. Wouldn't be the end of the world.